crates/transaction-pool/src/config.rs

```diff
@@ -36,6 +39,8 @@ pub struct PoolConfig {
     /// How to handle locally received transactions:
     /// [`TransactionOrigin::Local`](crate::TransactionOrigin).
     pub local_transactions_config: LocalTransactionConfig,
+    /// additional tasks size
+    pub additioanl_tasks_size: usize,
```

Comment:
this is not necessarily a poolconfig setting, I'd like to only keep this on the TxArgs because for the pool itself this isn't required